docs/source/release/v6.7.0/framework.rst

```diff
@@ -5,6 +5,22 @@ Framework Changes
 .. contents:: Table of Contents
    :local:
 
+Event Filtering
+---------------
+
+During the v6.7 development cycle, there were major changes to the code underlying event filtering that was originally designed `here <https://github.com/mantidproject/mantid/issues/34794>`_.
+The main changes to mantid are as followed:
+
+1. A ``TimeROI`` object that contains a list of use and ignore regions in time. The times are ``[inclusive, exclusive)``
+2. When filtering/splitting logs in the ``Run`` object, the logs are copied as is and a new ``TimeROI`` object is set on the ``Run``. The main change is that logs no longer have fake values to mimic the filtering
+3. Code should change from asking a ``TimeSeriesProperty`` for its statistics to asking the ``Run`` object for the statistics of a log because the ``Run`` is the only place that knows the information about the ``TimeROI``
+4. When filtering/splitting logs, the filters are now ``[inclusive, exclusive)`` where previous behavior was ``[inclusive, inclusive]``. The previous behavior was inconsistent with how events are filtered. This change, fixes issues observed with the integrated proton charge
+5. When workspaces are added, the resulting ``TimeROI`` is the union of the individual ``TimeROI``. If either workspace had an empty ``TimeROI`` it is assumed to be from that workspace's start-time to end-time.
```

Comment:
@jmborr , is the word "empty" wrt ``TimeROI`` used correctly in this context?

Reply:
@bolotovskyr I was talking to Pete about a very similar issue. I think the wording should be "If either workspace had a  ``TimeROI``  initialized as ``USE_ALL``, it is assumed to be from that workspace's start-time to end-time".

Reply:
We ended up changing that method name in #35424. I will update the PR shortly.